widget_driver_annotation/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: widget_driver_annotation
 description: Defines the annotations used by widget_driver and widget_driver_generator to create code for your WidgetDrivers
-version: 1.0.3
+version: 1.0.4
```

Comment:
You are not really changing the `widget_driver_annotation` package right? So I guess you don't have to bump this version?